config.toml

```diff
@@ -15,6 +15,7 @@ pluralizeListTitles = false
   keywords = ["eclipse", "research"]
   styles = "css/styles.css"
   gcse = "011805775785170369411:p3ec0igo0qq"
+  # hugo_js = "js/solstice.js"
```

Comment:
why add this line?

Reply:
Oops, btw, for this site it's also showing the err of "Not found", I'm thinking of adding a new empty one hugo_js = ""

Reply:
Ah, I think it only appears on localhost, I don't need it, Sorry